.github/workflows/ci.yaml

```diff
@@ -7,6 +7,11 @@ on:
     - staging
     - trying
 
+env:
```

Comment:
Wait, does this actually work? Docs don't say it does. Could you add a warning to rustc code to check this?

Reply:
checked on ra, it does work :tada: